webapp/packages/core-resource/src/Resource/CachedResourceOffsetPageKeys.ts

```diff
@@ -114,39 +114,48 @@ export function limitOffsetPages(pages: IResourceOffsetPage[], limit: number): I
   return result;
 }
 
```

Comment:
really hard to read what is going on here. please at least TODO to make it more readable

Reply:
thanks, removed (it's not in use anymore)